addons/hr_recruitment/static/src/js/tours/hr_recruitment.js

```diff
@@ -33,7 +34,8 @@ tour.register('hr_recruitment_tour',{
     trigger: ".o_job_alias",
     extra_trigger: '.o_hr_job_simple_form',
     content: _t("Choose an application email."),
-    position: "right"
+    position: "right",
+    width: 195
```

Comment:
I'm not sure it's a good idea to hardcode a width... What happens if a translation is much longer than the English content?